components/layout_2020/flow/line.rs

```diff
@@ -405,6 +405,9 @@ impl InlineBoxLineItem {
             content_rect.start_corner += &relative_adjustement(&style, state.ifc_containing_block);
         }
 
+        // NB: There is no need to set a baseline offset for this BoxFragment, because the
+        // baselines of this InlineFormattingContext is what will propagate to `display:
+        // inline-block` ancestors.
```

Comment:
should we use `///` doc comments here?

Reply:
I don't think so. rustdoc comments are only processed when they document rust functions, modules, and globally scoped variables